src/frontend/apps/lti_site/components/App/AppContentLoader/index.spec.tsx

```diff
@@ -91,7 +93,7 @@ describe('<AppContentLoader />', () => {
 
     await screen.findByText('content');
 
-    expect(decodeJwt).toHaveBeenCalledTimes(1);
+    expect(decodeJwt).toHaveBeenCalledTimes(2);
```

Comment:
I don't understand why it is called twice. And you should test `mockedDecodeJwt` I think, it was already `decodeJwt` but I think it's a mistake

Reply:
It is called twice because of the condition in AppContent and the one in AppContentLoader. But I need to change this test since we use useJwt.getState().getDecodedJwt() now

Reply:
`decodeJwt` is still call in this function and because we are saving the decoded token in the state, it should be called once. Or there is a bug in the `useJwt` store.